processing/src/main/java/org/apache/druid/math/expr/BuiltInExprMacros.java

```diff
@@ -214,4 +237,31 @@ public Object getLiteralValue()
       }
     }
   }
+
+  /***
+   * Alias Expression macro create an alias and deligates operations to same base macro
+   */
+  public static class AliasExprMacro implements ExprMacroTable.ExprMacro
```

Comment:
Is there a need to make this static? This can exist as a separate public class maybe?

Reply:
moved to new public class

Reply:
An internal class of `ExprMacroTable` would also probably be a more appropriate place for this to live so it is more obvious that functions defined by extensions could also use this